#linear velocities
linear_x: 3.5
linear_y: 0
linear_z: 0
#angular velocities
angular_x: 0
angular_y: 0
angular_z: 0.5